### DISCUSSION TITLE
Using Google 3D Tiles with GoogleMapsOverlay

### DISCUSSION THREAD

**Discussion by fjacupar** [visgl/deck.gl#9112] [Q&A]
Hello all, 

I just wanted to know if it is possible to use a Tile3DLayer from Google Maps, Cesium or ArcGIS with or inside an Overlay like GoogleMapsOverlay for example, or is it better to use with Deck standalone like the normal way.

Thanks in advance.

**Comment by ibgreen** [comment]
The major unsolved problem is to match the elevations in the 3d tiles with the basemap, they tend to float above.

**Reply by ronhasson** [reply]
Hey, sorry to disturb you after a year. Any chance there is a solution to the elevation issue? It's really hard to find something that works
